tests/test_adapters_reddit.py

```
@@ -0,0 +1,103 @@
+
+
+@pytest.mark.asyncio
+async def test_deleted_author_handled():
+    adapter = RedditApiAdapter(
+        instance_id="r", scope="community/x", subreddit="X",
+    )
+    with patch.object(adapter, "_http_get",
+                      new=AsyncMock(return_value=_SAMPLE_REDDIT_JSON)):
+        alerts, _ = await adapter.fetch(None)
+
+    deleted_author_alert = [a for a in alerts if a.event_id == "t3_post2"][0]
```

Ruff (0.16.2)

[warning] Prefer `next(a for a in alerts if a.event_id == "t3_post2")` over single element slice

Replace with `next(a for a in alerts if a.event_id == "t3_post2")`

(RUF015)